business_requirement/models/business.py

```diff
@@ -0,0 +1,444 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import except_orm
+from openerp import tools
+from openerp import SUPERUSER_ID
+
+
+class BusinessRequirement(models.Model):
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _name = "business.requirement"
+    _description = "Business Requirement"
+    _order = 'id desc'
+
+    @api.model
+    def _get_default_company(self):
+        company_id = self.env.user._get_company()
+        if not company_id:
+            raise except_orm(
+                _('Error!'),
+                _('There is no default company for the current user!'))
+        return self.env['res.company'].browse(company_id)
+
+    sequence = fields.Char(
+        'Sequence',
+        readonly=True,
+        copy=False,
+        index=True,
+    )
+    name = fields.Char(
+        'Name',
+        required=False,
+        readonly=True,
+        copy=False,
```

Comment:
When you drag and drop a requirement using the handle, a js component will re-assign numbers for all requirements that have been loaded on that domain. That's why it's important to make sure that when you reorder, you do it in the appropiate domain (e.g. project). 

Not sure how you'd use the sequence field in the Kanban view.